home/tests/test_views.py

```diff
@@ -36,61 +68,53 @@ def test_page_rating_success(self):
 
         response_data = response.json()
         response_data.pop("timestamp")
         rating = ContentPageRating.objects.last()
-        self.assertEqual(
-            response.json(),
-            {
-                "id": rating.id,
-                "helpful": False,
-                "comment": "lekker comment",
-                "data": {"contact_uuid": "123"},
-                "page": page.id,
-                "revision": page.get_latest_revision().id,
-            },
-        )
-
-    def test_page_rating_required_fields(self):
-        user = get_user_model().objects.create_user("test")
-        self.client.force_authenticate(user)
 
-        response = self.client.post(self.url, {}, format="json")
+        assert response.json() == {
+            "id": rating.id,
+            "helpful": False,
+            "comment": "lekker comment",
+            "data": {"contact_uuid": "123"},
+            "page": page.id,
+            "revision": page.get_latest_revision().id,
+        }
 
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-        self.assertEqual(
-            response.json(),
-            {
-                "helpful": ["This field is required."],
-                "page": ["This field is required."],
-                "revision": ["This field is required."],
-            },
-        )
-
-    def test_page_rating_invalid_page(self):
-        user = get_user_model().objects.create_user("test")
-        self.client.force_authenticate(user)
+    def test_page_rating_required_fields(self, api_client):
+        """
+        Ensure that the helpful, page and revision fields are required
+        and rating is not created when these fields are missing
+        """
+        response = api_client.post("/api/v2/custom/ratings/", {}, format="json")
 
-        response = self.client.post(self.url, {"page": 123}, format="json")
+        assert response.status_code == status.HTTP_400_BAD_REQUEST
+        assert response.json() == {
+            "helpful": ["This field is required."],
+            "page": ["This field is required."],
+            "revision": ["This field is required."],
+        }
 
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-        self.assertEqual(
-            response.json(),
-            {
-                "page": ["Page matching query does not exist."],
-            },
+    def test_page_rating_invalid_page(self, api_client):
+        """
+        Ensure that a rating cannot be made for a page that does not exist
+        """
+        response = api_client.post(
+            "/api/v2/custom/ratings/", {"page": 123}, format="json"
         )
 
-    def test_get_list(self):
+        assert response.status_code == status.HTTP_400_BAD_REQUEST
+        assert response.json() == {
+            "page": ["Page matching query does not exist."],
+        }
+
+    def test_get_list(self, api_client):
         """
         Should return the data, filtered by the querystring
```

Comment:
Which data? Maybe something like "The list endpoint returns all ratings that match the query filter" for this test?